src/main/java/edu/harvard/iq/dataverse/PermissionServiceBean.java

```diff
@@ -50,347 +53,392 @@
 public class PermissionServiceBean {
 
     private static final Logger logger = Logger.getLogger(PermissionServiceBean.class.getName());
-    
+
+    /**
+     * A set of permissions that con only be granted to {@link AuthentictedUser}s.
+     */
     private static final Set<Permission> PERMISSIONS_FOR_AUTHENTICATED_USERS_ONLY = 
-           EnumSet.copyOf(Arrays.asList(Permission.values()).stream()
-                    .filter( Permission::requiresAuthenticatedUser )
-                    .collect( Collectors.toList() ));
-    
+        EnumSet.copyOf(Arrays.asList(Permission.values()).stream()
```

Comment:
Why copy?

Reply:
Because we want the actual `Set` object to be an `EnumSet`, not just any set. This means set operations are done over bit of a `long` value, rather than over objects.